crates/fhir-sdk/tests/client-r4b.rs

```diff
@@ -23,13 +23,71 @@ use fhir_sdk::{
 	Date,
 };
 use futures::TryStreamExt;
+use reqwest::header::HeaderValue;
+use serde_json::json;
+use tokio::sync::OnceCell;
+
+fn extract_json_field(json_body: serde_json::Value, field: &str) -> Option<String> {
+	let code = json_body.as_object()?.get(field)?.as_str()?;
+	Some(code.to_owned())
+}
+
+async fn medplum_auth() -> Result<HeaderValue> {
```

Comment:
R4B should be tested against an R4B server, so it needs to stay the HAPI one right?

Reply:
Yes forgot to remove it.